propagators/opentelemetry-propagator-jaeger/test/JaegerHttpTracePropagator.test.ts

```diff
@@ -189,21 +202,36 @@ describe('JaegerHttpTracePropagator', () => {
     const ctx1 = jaegerHttpTracePropagator.extract(
       ROOT_CONTEXT,
       carrier,
-      (c, k) => 1 // not a number
+      makeGetter(1) // not a number
     );
     const ctx2 = jaegerHttpTracePropagator.extract(
       ROOT_CONTEXT,
       carrier,
-      (c, k) => [] // empty array
+      makeGetter([]) // empty array
     );
     const ctx3 = jaegerHttpTracePropagator.extract(
       ROOT_CONTEXT,
       carrier,
-      (c, k) => undefined // missing value
+      makeGetter(undefined) // missing value
     );
 
     assert.ok(ctx1 === ROOT_CONTEXT);
     assert.ok(ctx2 === ROOT_CONTEXT);
     assert.ok(ctx3 === ROOT_CONTEXT);
   });
 });
+
+function makeGetter(value: any) {
+  const getter: TextMapGetter = {
+    get(carrier, key) {
+      return value;
+    },
+    keys(carrier) {
+      if (carrier == null) {
```

Comment:
also here, better to use `===` In my opinion

Reply:
keys is exactly the same as `defaultTextMapGetter` https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-api/src/context/propagation/TextMapPropagator.ts#L124